File: cscms/go/ID_2_Remove_String.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var str_1, str_2 string
	fmt.Scan(&str_1)
	fmt.Scan(&str_2)
	var str_1_length = len([]rune(str_1))
	var str_2_length = len([]rune(str_2))

	if str_1_length > str_2_length {
		fmt.Print(strings.ReplaceAll(str_1, str_2, ""))
	} else {
		fmt.Print(strings.ReplaceAll(str_2, str_1, ""))
	}
}

File: cscms/java/ID_2_Remove_String.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str_1 = sc.nextLine();
        String str_2 = sc.nextLine();
        String result = "";

        if (str_1.length() > str_2.length()) {
            result = str_1.replaceAll(str_2, "");
        } else {
            result = str_2.replaceAll(str_1, "");
        }
        System.out.println(result);
        sc.close();
    }
}